setup.py

```diff
@@ -59,6 +59,7 @@
     'webcolors==1.7',
     # Invenio dependencies
     'Flask>=1.0.2',
+    'Flask-Login<0.5.0',
```

Comment:
I'm just testing locally the same fix :wink: 

Please leave a ` # FIXME waiting for Flask-Security release` after the pin, so that we don't forget